apps/openmw/mwmechanics/aifollow.cpp

```diff
@@ -27,9 +27,9 @@ struct AiFollowStorage : AiTemporaryBase
 
 int AiFollow::mFollowIndexCounter = 0;
 
-AiFollow::AiFollow(const std::string &actorId,float duration, float x, float y, float z)
+AiFollow::AiFollow(const std::string &actorId,float duration, float x, float y, float z, bool repeat)
 : mAlwaysFollow(false), mCommanded(false), mRemainingDuration(duration), mX(x), mY(y), mZ(z)
-, mActorRefId(actorId), mActorId(-1), mCellId(""), mActive(false), mFollowIndex(mFollowIndexCounter++)
+, mActorRefId(actorId), mActorId(-1), mCellId(""), mActive(false), mFollowIndex(mFollowIndexCounter++), mRepeat(repeat)
 {
```

Comment:
For example, this constructor would become:
`AiFollow::AiFollow(const std::string &actorId,float duration, float x, float y, float z, bool repeat)
: AiPackage(repeat), mAlwaysFollow(false), mCommanded(false), mRemainingDuration(duration), mX(x), mY(y), mZ(z)
, mActorRefId(actorId), mActorId(-1), mCellId(""), mActive(false), mFollowIndex(mFollowIndexCounter++)`
You would then assign mRepeat in the AiPackage constructor